boot/freeldr/freeldr/include/ui.h

```diff
@@ -202,7 +202,7 @@ BOOLEAN    UiEditBox(PCSTR MessageText, PCHAR EditTextBuffer, ULONG Length);
 UCHAR    UiTextToColor(PCSTR ColorText);                        // Converts the text color into it's equivalent color value
 UCHAR    UiTextToFillStyle(PCSTR FillStyleText);                // Converts the text fill into it's equivalent fill value
 
-VOID    UiFadeInBackdrop(VOID);                                    // Draws the backdrop and fades the screen in
+VOID    UiFadeInBackdrop(ULONG DrawHeight);                     // Draws the backdrop and fades the screen in
```

Comment:
Since "FadeInBackdrop" is for clearing _all_ the screen, I would suggest you keep the original function signature, and, in the implementation of the function itself, you call the "DrawBackdrop" with the correct argument.

Reply:
Done